Zero-Gravity-Store-React/src/Pages/Products/Components/Cards.js

```diff
@@ -14,9 +14,10 @@ export function Cards({ product }) {
   return (
     <div className="card-vertical-box flex card-border">
       <div className="vertical-card flex">
-        <div className="card-top">
+        <Link to={`/products/${product._id}`}><div className="card-top">
```

Comment:
here the link tag is not proper. Either put a link tag around the img or the div